src/material-experimental/mdc-radio/radio.scss

```diff
@@ -70,6 +69,12 @@
     .mat-ripple-element {
       opacity: mdc-radio-theme.$ripple-opacity;
     }
+
+    // Radio components have to set `border-radius: 50%` in order to support density scaling
+    // which will clip a square focus indicator so we have to turn it into a circle.
+    &::before {
```

Comment:
I saw this a couple of times now. OOC, is this strictly related to the focus indicators? You mention density scaling, that's why I ask.

Reply:
I didn't write it, but I think that it's referring to the fact that it uses percentages instead of pixels for the radius.